### ISSUE TITLE
Unavailable

### ISSUE THREAD

**Issue by petertseng** [exercism/rust#349]
Personal issues. I don't know when I'll be back. It might be the case that I will update in a month. It is possible this will be permanent, or not.

**Comment by ijanos** [comment]
Ok! Take your time and take care, you are always welcome back.

**Comment by petertseng** [comment]
Somewhat available, as time permits. Don't count on me.

**Comment by petertseng** [comment]
So, the current deal now is that things like README generation and nextercism increase the amount of work needed per-track. I have already had to completely drop one of the four tracks I used to maintain (it is thankfully in very good hands now). Of the three remaining tracks, I'm sorry but Rust is going to have to take the lowest priority because there are other maintainers. I'm all alone in my two other tracks (Haskell, Ceylon). Usually load is low but right now not so, so I'm very behind on Rust. So, don't count on me still.
